Challenge/View/Movies/MovieDetail/MovieDetailView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  MovieDetailView.swift
+//  Challenge
+//
+//  Created by Lucas Henrique Machado da Silva on 06/05/2018.
+//  Copyright © 2018 Lucas Henrique Machado da Silva. All rights reserved.
+//
+
+import UIKit
+
+class MovieDetailView: UIViewController {
+    // MARK: - Outlets
+    @IBOutlet weak var tvDetail: UITableView!
+    
+    // MARK: - Presenter
+    var presenter: MovieDetailPresenter?
+    
+    // MARK: - Variables
+    var movie: Movie?
```

Comment:
@arctouch-ruannhomem I use a local movie variable on my view because it can be different from the presenter one, presenter will always be with all movies obtained from the service, view will be only with the ones that will be viewed by the user (for example when user search for something).